src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), BEM block modified should be used after the block itself. For example, `calendar calendar--start-day-sun calendar--month-length-31` should be changed to `calendar calendar--31 calendar--start-day-sun`.